nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/features/PlatformFunctions.java

```diff
@@ -69,6 +73,10 @@ public DirectoryService getDirService() {
 
     public String getVocabularyLabel(String voc, String key) {
         try (Session session = getDirService().open(voc)) {
+            if (!session.hasEntry(key)) {
+                log.warn("Unable to find the key '" + key + "' in the vocabulary '" + voc + "'.");
```

Comment:
Not sure we want a WARN here, this could spam the logs and the server admin can't do much about it. I'd do just DEBUG.